common/Resources/ti.internal/bootstrap.loader.js

```diff
@@ -112,14 +103,13 @@ exports.loadAsync = function (finished) {
 
 	// Loads all bootstrap scripts found.
 	function loadBootstrapScripts(finished) {
-		var bootstrapIndex = 0;
+		let bootstrapIndex = 0;
 		function doLoad() {
 			// Attempt to load all bootstrap scripts.
-			var fileName, bootstrap;
 			while (bootstrapIndex < bootstrapScripts.length) {
 				// Load the next bootstrap.
-				fileName = bootstrapScripts[bootstrapIndex];
-				bootstrap = require(fileName);
+				const fileName = bootstrapScripts[bootstrapIndex];
+				const bootstrap = require(fileName);
```

Comment:
<!--
  0 failure: 
  1 warning:  common/Resources/...
  
  
  DangerID: danger-id-default;
  File: common/Resources/ti.internal/bootstrap.loader.js;
  Line: 112;
-->

- :warning: common/Resources/ti.internal/bootstrap.loader.js line 112 – Found non-literal argument in require (security/detect-non-literal-require)